meta-ostro/conf/distro/include/ostroproject-ci.inc

```diff
@@ -0,0 +1,2 @@
+# XXX Drop the VM hack after taking care also of the non UEFI devices (those using U-Boot: edison and beaglebone)
+OSTRO_VM_IMAGE_TYPES = "dski.zip dsk.vdi.zip"
```

Comment:
On Fri, 2016-03-18 at 03:19 -0700, Igor Stoppa wrote:


> ​Doesn't CI need to produce also images based on
> ostro-os-production.inc?​ ​Wouldn't the inclusion here of ​​
> ostro-development.inc here prevent those?

It is impossible to produce images for both production and development
in the same build configuration; the CI exclusively configures itself to
build development images in all jobs.

If we want to change that (I believe we should, at least to ensure that
a production configuration builds, but that's a different story), we'd
have to add a "ostroproject-production-ci.inc" that gets included in
different build jobs and re-arrange our .inc files accordingly.